build/global-styles

```diff
@@ -8,5 +8,6 @@ const {
   createBaseStyles
 } = require('../src/components/BaseStyles/BaseStylesService');
 
-const globalStyles = createBaseStyles(circuit);
-console.log(format(globalStyles, { parser: 'css' }));
+const { styles } = createBaseStyles(circuit);
+
+console.log(format(styles, { parser: 'css' }));
```

Comment:
FYI, this is legacy code that is no longer maintained. It will be removed in #479.

Reply:
yes, I know, but it failed and prevented to test build commands, so I decided to fix it as it was quite clear what's wrong here :)